medicines/docs/data/README.md

```diff
@@ -0,0 +1,81 @@
+# Data
+
+Details are included here of how the data that powers the products site is stored and updated.
+
+## Overview
+
+The main data used by the products site relates predominantly to 3 types of document:
+
+- Summaries of Product Characteristics (SPC)
```

Comment:
suspect this is a direct copy-paste but it's odd that _Summaries_ is plural and _Leaflet_ and _Report_ below are not

Reply:
It is indeed odd - maybe because each document features more than one summary?

Reply:
Suspect you're right. These are the names we're using on the frontend.